__tests__/app.test.js

```diff
@@ -0,0 +1,25 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+
+afterAll(() => {
+  return db.end();
+});
+
+
+// 1. GET /api/topics
+
+describe("GET /api/topics", () => {
+  test("1. GET 200: Returns an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.topics).toBeInstanceOf(Array);
+        response.body.topics.forEach((topic) => {
```

Comment:
As we're using a forEach here, it would be great if we could check the length of the array beforehand 👍